DataRepo/models.py

```diff
@@ -319,6 +323,85 @@ class PeakGroup(models.Model):
     def atom_count(self, atom):
         return atom_count_in_formula(self.formula, atom)
 
+    @cached_property
+    def total_abundance(self):
+        """
+        Total ion counts for this compound. Accucor provides this in the tab
+        "pool size". Calculated by summing the corrected_abundance of all
+        Measurements for this compound.
+        """
+        return self.peak_data.all().aggregate(
+            corrected_abundance=Sum("corrected_abundance")
+        )["corrected_abundance"]
```

Comment:
I was uncertain of the logic, since I have not used `aggregate` , but would this be the same?  Mostly, I want to understand this wrapped line.

```suggestion
        return self.peak_data.all().aggregate(
            corrected_abundance_sum=Sum("corrected_abundance")
        )["corrected_abundance_sum"]
```

Reply:
Yes, that would be the same.